solidstart/username-and-password/package.json

```diff
@@ -7,19 +7,17 @@
 		"start": "node ./.output/server/index.mjs"
 	},
 	"dependencies": {
-		"@lucia-auth/adapter-sqlite": "^3.0.0",
-		"@solidjs/router": "^0.10.5",
-		"@solidjs/start": "^0.4.2",
-		"better-sqlite3": "^9.2.2",
-		"lucia": "^3.0.1",
-		"oslo": "^1.0.3",
-		"solid-js": "^1.8.7",
-		"vinxi": "0.0.54"
+		"@lucia-auth/adapter-sqlite": "^3.0.1",
+		"@solidjs/router": "^0.13.3",
+		"@solidjs/start": "^1.0.0",
+		"@types/better-sqlite3": "^7.6.10",
```

Comment:
Again, why are types in regular dependencies? They're not required at runtime, only during development.